examples/ex5p.cpp

```diff
@@ -317,6 +317,7 @@ int main(int argc, char *argv[])
    VisItDataCollection visit_dc("Example5-Parallel", pmesh);
    visit_dc.RegisterField("velocity", u);
    visit_dc.RegisterField("pressure", p);
+   // visit_dc.SetFormat(1); // save the mesh using MFEM's parallel mesh format
```

Comment:
Given that this is user facing we might want to start an enum with different format names to use.  Or if what we really want just a bool then we should rename the method to something like SetFormatParallel

Reply:
Yes, we can do that (I was just being lazy). Note that the formats will depend on the derived classes, so the enumerations should be defined there. The `SetFormat()` method in the base class can still accept `int` argument to support all values from different derived classes.

How does this sound?

Reply:
Done in 83dd511027 for the base class, `DataCollection`, which should also work for `VisitDataCollection`. The `SidreDataCollection` class has its own `Save()` method where the format can be specified as a string.